File: src/design/utilities/GameSettings.java
package design.utilities;

public interface GameSettings {
	
	final static int WIDTH = 1280;
	final static int HEIGHT = 768;
	final static int TILESIZE = 64;
	final static int TOTALTILES = ((WIDTH-TILESIZE)*(HEIGHT-TILESIZE))/(TILESIZE*TILESIZE);
	
	final static int MINDOORS = 1;
	final static int MAXDOORS = 4;
	
	final static int LIMITRIGHT = WIDTH - TILESIZE;
	final static int LIMITLEFT = TILESIZE - 1;
	final static int LIMITUP = TILESIZE - 1;
	final static int LIMITDOWN = HEIGHT - TILESIZE;
	
	
}

File: Ragazzini_Marco/CSharp_Task/DynamicBody/GameSettings.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Model.DynamicBody
{
	public static class GameSettings
	{
		public enum Value
        {
			Width = 1280,
			Height = 768,
			Tilesize = 64,

			LimitRight = Width - Tilesize,
			LimitLeft = Tilesize - 1,
			LimitUp = Tilesize - 48 - 1,
			LimitDown = Height - Tilesize
		}
    }
}
